Marlin/temperature.cpp

```diff
@@ -203,7 +203,7 @@ uint8_t Temperature::soft_pwm[HOTENDS];
   int Temperature::current_raw_filwidth = 0;  //Holds measured filament diameter - one extruder only
 #endif
 
-#ifdef ADC_KEYPAD
+#if ENABLED(ADC_KEYPAD) && PIN_EXISTS(ADC_KEYPAD)
```

Comment:
The thing is I do not have a ADC_KEYPAD, so I believe the output file size is different.
see for the why, also my sanity check, in the PR #5987
Maybe I do not complete understand why the sanity check fail, but the pin is not defined.